Sources/xcodeproj/Extensions/NSRecursiveLock+Sync.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension NSRecursiveLock {
+    func sync<T>(closure: () -> (T)) -> T {
```

Comment:
Is this being used?

Reply:
I saw `whileLocked()` naming across SPM in similar extensions. Perhaps, you might consider using it as well.

Reply:
That's a good idea.